docs/how-to-provision-using-dps.md

```diff
@@ -19,58 +19,64 @@ ms.service: digital-twins
 
 In this article, you'll learn how to integrate Azure Digital Twins with [Device Provisioning Service (DPS)](https://docs.microsoft.com/en-us/azure/iot-dps/about-iot-dps).
 
-The solution described in this article will allow you to automate the process to _provision_ and _retire_ IoT Hub devices in Azure Digital Twins using Device Provisioning Service. The better understand a set of general device management stages that are common to all enterprise IoT projects, see [Device Lifecycle](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-management-overview#device-lifecycle).
+The solution described in this article will allow you to automate the process to _provision_ and _retire_ IoT Hub devices in Azure Digital Twins, using Device Provisioning Service. 
+
+For more information about the _provision_ and _retire_ stages, and to better understand the set of general device management stages that are common to all enterprise IoT projects, see the [*Device Lifecycle* section](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-management-overview#device-lifecycle) of IoT Hub's device management documentation.
 
 ## Prerequisites
 
-Before you can set up the provisioning, you need to have an **Azure Digital Twins instance**. 
+Before you can set up the provisioning, you need to have an **Azure Digital Twins instance**.
 
-If you do not have this set up already, you can create it by following the Azure Digital Twins [*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md). The tutorial will walk you through setting up an Azure Digital Twins instance and an Azure IoT Hub.
+If you do not have this set up already, you can create it by following the Azure Digital Twins [*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md). The tutorial will walk you through setting up an Azure Digital Twins instance and a connected Azure IoT Hub.
 
```

Comment:
I think we can be a little more specific about the requirement here, to make sure they only do what they need and no more --

If all they need is an Azure Digital Twins instance, they can follow the simpler [setup article](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-set-up-instance-scripted) instead of the end-to-end tutorial.

If they need the instance to contain models and twins, they only have to complete the end-to-end tutorial through [instantiating the pre-created graph](https://docs.microsoft.com/en-us/azure/digital-twins/tutorial-end-to-end#instantiate-the-pre-created-twin-graph).

If they need the instance to contain models and twins and have a published Azure function app, they only have to complete the end-to-end tutorial through [set up the sample function app](https://docs.microsoft.com/en-us/azure/digital-twins/tutorial-end-to-end#set-up-the-sample-function-app).

If they need the twins to be connected so that they update with sample data from a simulated device, *then* they should complete the entire [end-to-end tutorial](https://docs.microsoft.com/en-us/azure/digital-twins/tutorial-end-to-end).

Can you clarify which one is required in the introductory prerequisite sentence, and adjust the link if necessary?

Reply:
We need all, and our sample extends the end-to-end scenario, including ADT instance, models & twins, Function App and IoT Hub with ingestion flow. So I think we should keep the reference to the end-to-end tutorial.